backend/papersfeed/utils/recommendations/utils.py

```diff
@@ -22,17 +27,33 @@ def select_user_actions(_):
         'UserId', 'ItemId', 'Type', 'Count'
     )
 
-    for action in new_actions:
-        paper_id = action['ItemId']
+    new_papers = Paper.objects.filter(
+        creation_date__gt=(datetime.now() + timedelta(days=-1))
+    ).annotate(
+        ItemId=F('id')
+    ).values(
+        'ItemId'
+    )
+
+    new_users = User.objects.all().annotate(
```

Comment:
Why does this code not filter with `creation_date` of users?

Reply:
Because user model does not inherit BaseModel.

Reply:
So user models do not have creation_date or modification_date ㅠㅠ

Reply:
Ah, got it.